### DISCUSSION TITLE
[Request] Pin dashboards on top of the dashboard list view

### DISCUSSION THREAD

**Discussion by lctdulac** [apache/superset#18455] [Ideas]
Hi SuperSet community,
As our use of SuperSet is growing and we have more and more dashboards in the system, it is becoming harder to see clearly what are **the most important dashboards.**
I propose a pin feature in the dashboard list, to highlight most important dashboards first.

![Screenshot 2021-03-05 at 12 10 09](https://user-images.githubusercontent.com/46674763/110108116-1380ff00-7dac-11eb-9501-09dfe2d6c2c4.png)

What do you guys think? Would that be something you would be interested as well ?

**Comment by junlincc** [comment]
Hi @lctdulac, thanks for proposing. Does "star" not serve this purpose? Starred items naturally get on the top of the list. Let us know~

**Comment by lctdulac** [comment]
Hi @junlincc thanks for replying! Star does not automatically ping the dashboard on top. The user still need to filter based on stars. If an organization has say 100 dashboards, but one ''main'' one that should be used by everyone, it would make sense to be able to ''ping'' it on top of the dashboard list. What do you think ?

**Comment by junlincc** [comment]
@lctdulac I think the request is valid. To achieve your goal is only one more click away from staring the item though. I see how Pin can be useful when there are hundreds of Starred dashboard in thousands of dashboard in the list. :) deferring the question to our product designer. I will drop in design office hour next week to see if we can come up with some solutions. @steejay @mihir174

**Comment by lctdulac** [comment]
Cool, thanks all!
